mne/time_frequency/tfr.py

```diff
@@ -670,12 +670,20 @@ def plot(self, picks=None, baseline=None, mode='mean', tmin=None,
         from ..viz.topo import _imshow_tfr
         import matplotlib.pyplot as plt
         times, freqs = self.times.copy(), self.freqs.copy()
-        data = self.data[picks]
+        info = self.info
+        data = self.data
 
         data, times, freqs, vmin, vmax = \
             _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
                          baseline, vmin, vmax, dB)
 
+        if picks is None:
+            picks = pick_types(info, meg=True, eeg=True, ref_meg=False,
+                               exclude='bads')
+        picks = np.atleast_1d(picks)
```

Comment:
I am a bit worried by this atleast_1d? you expect people to pass a single int to picks?
to me it's an error that should not be passes silently